sim/midas/src/main/scala/midas/passes/BridgeTopWiring.scala

```diff
@@ -0,0 +1,157 @@
+package midas.passes
+
+import scala.collection.mutable
+
+import firrtl._
+import firrtl.analyses.InstanceGraph
+import firrtl.annotations._
+import firrtl.ir._
+import firrtl.Mappers._
+import firrtl.transforms.TopWiring.{TopWiringAnnotation, TopWiringTransform, TopWiringOutputFilesAnnotation}
+import TargetToken.{Instance, OfModule}
+
+import midas.passes.fame.RTRenamer
+
+
+case class BridgeTopWiringAnnotation(target: ReferenceTarget, clock: ReferenceTarget) extends Annotation {
```

Comment:
```suggestion
case class BridgeTopWiringAnnotation(target: ReferenceTarget, clock: ReferenceTarget) extends Annotation with FAMEAnnotation {
```

This will make them dump with the rest of the annotations in the debug JSON files.